bndtools.core/src/bndtools/central/RepositoriesViewRefresher.java

```diff
@@ -127,6 +128,12 @@ public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
 								busy = false;
 								if (redo) {
 									refreshRepositories(null);
+								} else {
+									try {
+										Central.refreshProjects();
+									} catch (Exception e) {
+										Exceptions.duck(e);
```

Comment:
This should be `throw Exceptions.duck(e);` for proper escape analysis by tools/compilers.